src/main/java/org/jenkinsci/plugins/nomad/NomadCloud.java

```diff
@@ -26,7 +26,7 @@
 
     private static final Logger LOGGER = Logger.getLogger(NomadCloud.class.getName());
     private final List<? extends NomadSlaveTemplate> templates;
-    private final String name;
+    protected final String name;
```

Comment:
I think this should be removed altogether, it's defined in the parent class and I don't think there's a special need to redefine it here?

Reply:
Removed.

Reply:
I haven't tested the fix but that looks good to me